pages/view/_iscnId.vue

```diff
@@ -227,5 +339,75 @@ export default class ViewIscnIdPage extends Vue {
       this.$nuxt.error({ statusCode: 404, message: ErrorMessage.statusCode404 })
     }
   }
+
+  get record() {
+    return this.getISCNById(this.iscnId)?.data
+  }
+
+  get metadata() {
+    return this.record && (this.record as any).contentMetadata
+  }
+
+  get type() {
+    return this.metadata && this.metadata['@type']
+  }
+
+  get imgSrc() {
+    return (
+      (this.type === 'Image' || this.type === 'Photo') &&
+      getIPFSUrlFromISCN(this.getISCNById(this.iscnId))
+    )
+  }
+
+  get name() {
+    return this.metadata.name || this.metadata.title
+  }
+
+  get keywords(): Array<string> {
+    return this.metadata.keywords ? this.metadata.keywords.split(',') : []
+  }
+
+  get stakeholders() {
+    return this.record.stakeholders
+  }
+
+  get likerIdURL(){
+     return LIKER_LAND_URL + this.stakeholderInfo.likerId
+  }
+
+  get transactionsURL() {
+    return BIG_DIPPER_TRANSACTIONS + this.txHash
+  }
+
+  get rawDataURL(){
+    return this.url + this.iscnId
+  }
+
+  async fetchTxHash() {
+    let txHash = await fetch(RAWDATA_URL_PRODUCTION + this.iscnId)
+      .then((res) => res.json())
+    if (!txHash.txs) {
+      txHash = await fetch(RAWDATA_URL_STAGING + this.iscnId)
+        .then((res) => res.json())
+        .then((res) => res.txs[0].txhash)
```

Comment:
Should filter by `{"key":"action","value":"create_iscn_record"}` since we could have other tx related to this iscn_id, should not hardcode `txs[0]`